packages/amazonq/.changes/next-release/Feature-fb724754-850b-41b3-b3dd-2484b40720ff.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Amazon Q feature dev: add feedback telemetry."
+}
```

```suggestion
{
	"type": "Feature",
	"description": "Amazon Q feature dev: add feedback telemetry."
}
```
Telemetry isn't user-facing, so we don't need a changelog item. But thanks for being pro-active about adding one!